src/components/ShellView/ResultTable.vue

```diff
@@ -40,20 +40,40 @@
             </th>
           </tr>
         </thead>
-        <tbody>
-          <tr v-for="(row, i) in rows" :key="i">
-            <td v-for="(cell, j) in row" :key="j">
-              <ul v-if="Array.isArray(cell)" class="list-group">
-                <li v-for="(item, k) in cell" :key="k" class="list-group-item">
-                  <b>{{ item.name }}:</b> {{ item.value }}
-                </li>
-              </ul>
-
-              <span v-else>{{ cell }}</span>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <tbody>
+      <tr v-for="(row, i) in rows" :key="i">
+        <td v-for="(cell, j) in row" :key="j">
+          <template v-if="Array.isArray(cell)">
+            <ul class="list-group">
+              <li v-for="(item, k) in cell" :key="k" class="list-group-item">
+                <b>{{ item.name }}:</b> {{ item.value }}
+              </li>
+            </ul>
+          </template>
+          <template v-else>
+            <div v-if="cell.hasOwnProperty('_nodes') && cell.hasOwnProperty('_rels')">
```

Comment:
This is not robust. You should use column type instead of relying on if it has `_nodes` and `_rels`.